src/main/java/io/vertx/core/datagram/impl/DatagramSocketImpl.java

```diff
@@ -80,6 +81,9 @@ private DatagramSocketImpl(VertxInternal vertx, DatagramSocketOptions options) {
     if (options.getLogActivity()) {
       channel.pipeline().addLast("logging", new LoggingHandler());
     }
+    if ((options.getPcapCaptureFile() != null) && !options.getPcapCaptureFile().isEmpty()) {
+      channel.pipeline().addLast("logging", new VertxPcapWriteHandler(options.getPcapCaptureFile()));
```

Comment:
I guess, handler name can be changed here to "pcapCapturing" ?

Reply:
Good catch, fixed!

Reply:
thanks @Unev